llm/vllm/vllm.sky.yaml

```diff
@@ -0,0 +1,42 @@
+# Serve Llama 4 with vLLM on SkyPilot.
+# 
+# Usage:
+#   sky launch vllm.sky.yaml -c vllm-llama4 --secret HF_TOKEN=YOUR_HUGGING_FACE_API_TOKEN
+#   sky serve up -n vllm-llama4 vllm.sky.yaml --secret HF_TOKEN=YOUR_HUGGING_FACE_API_TOKEN
+envs:
+  MODEL_NAME: meta-llama/Llama-4-Maverick-17B-128E-Instruct-FP8  # Replace with your huggingface model name
+
+secrets:
+  HF_TOKEN: null # Pass with `--secret HF_TOKEN` in CLI
+
+resources:
+  accelerators: { H100:8, H200:8, B100:8, B200:8, GB200:8 }
```

Comment:
can we use a smaller model with smaller GPUs for the single node example to make it more accessible?

Reply:
Also, could we use this model `NousResearch/Meta-Llama-3-8B-Instruct` to avoid the requirement of HF_TOKEN to keep the example very easy to start with: https://docs.vllm.ai/en/latest/serving/openai_compatible_server.html